.github/workflows/convention-develocity-maven-extension-verification.yml

```diff
@@ -62,11 +62,9 @@ jobs:
       - name: Set Maven version
         if: ${{ matrix.versions.version != '(Current)' }}
         working-directory: convention-develocity-maven-extension/examples/maven_${{ matrix.versions.sample }}
-        run: ./mvnw wrapper:wrapper -Dmaven=${{ matrix.versions.version }} -Dgradle.enterprise.url=https://ge.solutions-team.gradle.com
-        env:
-          DEVELOCITY_ACCESS_KEY: ${{ secrets.GE_SOLUTIONS_ACCESS_TOKEN }}
+        run: ./mvnw wrapper:wrapper -Dmaven=${{ matrix.versions.version }} -Ddevelocity.scan.disabled
```

Comment:
Why do we disable this scan?

Reply:
It's only setting the version of the wrapper so I don't think it's necessary. The Gradle workflow [already does this](https://github.com/gradle/develocity-build-config-samples/blob/0ad93f704f3ba006e363132b3e7fe8d8ef075792/.github/workflows/convention-develocity-gradle-plugin-verification.yml#L80-L81) so this just brings it in line with that. 

This has nothing to do with the sample. I only noticed it as an inconsistency while creating the new workflow and wanted to fix it.

Reply:
Works for me 👍

Reply:
FYI – I decided to break this out to a separate PR in case this PR takes a while to be merged. https://github.com/gradle/develocity-build-config-samples/pull/1318